apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html

```diff
@@ -30,15 +30,14 @@
                             </ng-template>
                         </sh-select>
                     </div>
-                    <div class="input-group">
-                        <span class="input-wrapper">
-                            <input type="text" name="dateRange" daterangepicker [options]="datePickerOptions"
-                                (applyDaterangepicker)="onDateChange($event)" class="form-control cursor-pointer" />
-                        </span>
+                    <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                        <i class="icon-calender calendar-addon"></i>
```

Comment:
sometimes svg is added and sometimes icon, please use 1 thing only at all places